specification.md

```diff
@@ -274,18 +282,18 @@ OpenGDPR service implementations **MUST** provide an endpoint that creates OpenG
 
 #### 7.1.2 Extensions
 
-OpenGDPR requests **MAY** contain an `extensions` object, composed of a series of child-objects, keyed by a processor domain.
+OpenCompliance requests **MAY** contain an `extensions` object, composed of a series of child-objects, keyed by a processor domain.
 
-- The domain of each extension **MUST** match the processor's OpenGDPR domain, matching the `X-OpenGDPR-Processor-Domain` header in OpenGDPR responses.
+- The domain of each extension **MUST** match the processor's OpenCompliance domain, matching the `X-OpenCompliance-Processor-Domain` header in OpenCompliance responses.
 - Extensions **MUST NOT** be used for or contain authentication information.
 - Processors **MUST** only implement an extension for items that do not already fit into the generic spec.
 
-[Currently known extensions can be found here](OpenGDPR_extensions.md).
+[Currently known extensions can be found here](OpenCompliance_extensions.md).
 
-### 7.2.  Example OpenGDPR Request
+### 7.2.  Example OpenCompliance Request
 
 ```http
-POST /opengdpr_requests HTTP/1.1
+POST /OpenCompliance_requests HTTP/1.1
```

Comment:
Maybe it's better to use only `/requests` as Endpoint. This would work for all DP regulations.

singular.net does this already: https://gdpr.singular.net/gdpr/requests

Reply:
I agree this is cleaner if the processor scopes the path with a subdomain or containing path (like Singular did).  All URLs must include the version number (3.4) so breaking changes should not be a problem. 

@kriskalish What do you think?

Reply:
@bhox The suggestion is valid. It is succinct and clear. Realistically all implementations will either live on a subdomain or use a route prefix which sets the context for what "requests" means.

Reply:
Agreed, updated pr incoming